entrypoint/main.go

```diff
@@ -52,55 +91,85 @@ func runDaemon() error {
 
 	log.Debug("starting actual daemon via exec(3)")
 
-	return syscall.Exec("/icingadb", []string{"/icingadb", "-config", config}, os.Environ())
+	return syscall.Exec("/icingadb", []string{"/icingadb", "-c", config, "--datadir", "/data"}, os.Environ())
```

Comment:
Should consistently use short or long flags.

Reply:
Why? Not this repo's problem if an upstream flag doesn’t have a short version.

Reply:
There's a long version for both. Why not go with that?

Reply:
Oh, c'mon. The next time I see you typing a tar command with many short flags and one long one, I'll say this to you.